src/modules/ROOT/pages/changelog/202303/05.json

```diff
@@ -63,7 +52,7 @@
   "fixed": [
     {
       "what": [
-        "overflowing of long texts in {APP_NAME_COMPOSE}."
+        "long texts to no longer overflow in {APP_NAME_COMPOSE}."
```

Comment:
On second thought, is this better off as a fix? Either works for me; pick and choose